src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx

```diff
@@ -59,7 +59,9 @@ Thr top of every doc begins with a set of metadata. Read on for more information
 
     <tr>
       <td>tags</td>
-      <td>Keywords related to your doc. Start each topic with a `_`* on a new line. 
+      <td>Keywords related to your doc. Through some technical wizardry, they generate *Related resources* links in the right nav area. 
```

Comment:
Ah, OK. I do think we need a MetaDescription definition in that doc, but since there are some ambiguities around it, there's no need to hold up your PR.